packages/apps/composer-app/src/App.tsx

```diff
@@ -47,19 +47,20 @@ export const App = () => {
       resourceExtensions={[osTranslations, appkitTranslations, composerTranslations]}
       fallback={<Fallback message='Loading...' />}
       appNs='composer'
+      tooltipProviderProps={{ delayDuration: 100, skipDelayDuration: 400, disableHoverableContent: true }}
     >
       <ErrorProvider>
         {/* TODO(wittjosiah): Hook up user feedback mechanism. */}
         <ErrorBoundary fallback={({ error }) => <FatalError error={error} />}>
           <ClientProvider config={configProvider} services={servicesProvider} fallback={ClientFallback}>
-            <HashRouter>
+            <BrowserRouter>
```

Comment:
I had to do this in order for `useSearchParams` to work. Not sure quite why, though I prefer `BrowserRouter` anyway.

Reply:
@richburdon this is an example of where the old hash routing method has got some cobwebs on it, and modern routing needs (such as `useSearchParams`) no longer play with that method nicely anymore.